app/controllers/api/v1/recipes_controller.rb

```diff
@@ -53,7 +53,9 @@ def set_recipe
       # Only allow a list of trusted parameters through.
       def recipe_params
         params.require(:recipe)
-              .permit(:name, :description, :servings, :prep_time, :cook_time, :sprouty_pie, :lollys_pantry)
+              .permit(:id, :name, :description, :servings, :prep_time,
+                      :cook_time, :sprouty_pie, :lollys_pantry,  :created_at, :updated_at,
+                      ingredients_attributes: [:id, :recipe_id, :quantity, :measurement, :name, :preparation, :created_at, :updated_at, :_destroy,])
```

Comment:
Is it possible to get this from Ingredients controller. I think this array is duplicated between them.

Reply:
nope, this is allowlisting the params for this controller. it's where it needs to b